lib/utils.pm

```diff
@@ -484,6 +488,7 @@ for example:
 C<dumb_term> will default to C<is_serial_terminal()>.
 =cut
 sub zypper_call {
+    $IN_ZYPPER_CALL = 1;
```

Comment:
please move this further down below the `die 'Exit code…'` so that we set this a bit nearer to where we would actually be within a real zypper operation